From 865f672f7309a5d74afdd421f526a367c9bef430 Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Thu, 2 Jul 2015 12:08:02 +0200
Subject: [PATCH 018/100] Fix ich9-intel-hda compatibility

Message-id: <1435838882-10566-3-git-send-email-dgilbert@redhat.com>
Patchwork-id: 66604
O-Subject: [RHEL-7.2 qemu-kvm-rhev PATCH 2/2] Fix ich9-intel-hda compatibility
Bugzilla: 1209793
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

The MSI address for intel-hda got moved (somewhere around 2.2 ?)
by d209c7440a642ba08bbb0f13e22390460d3661ed which added
 a compat entry for 'intel-hda' to keep the MSI address the same.
Unfortunately that only moved 'intel-hda' and not 'ich9-intel-hda'.
This changes the compat entry to 'intel-hda-generic' to catch
both cases.

This is unfixable upstream, since that would break migrations between
2.2->2.4 while fixing pre-2.2->2.4.  Fortunately we never shipped 2.2.

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/compat.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/hw/compat.h b/include/hw/compat.h
index 88076f5..ec5d0e6 100644
--- a/include/hw/compat.h
+++ b/include/hw/compat.h
@@ -37,7 +37,7 @@
  */
 #define HW_COMPAT_RHEL7_1 \
         {\
-            .driver   = "intel-hda",\
+            .driver   = "intel-hda-generic",\
             .property = "old_msi_addr",\
             .value    = "on",\
         },{\
-- 
1.8.3.1

